packages/nvidia_nat_langchain/src/nat/plugins/langchain/langgraph_workflow.py

```
@@ -0,0 +1,164 @@
+        try:
+            output = await self._graph.ainvoke(value.model_dump())
+            return LanggraphWrapperOutput.model_validate(output)
+        except Exception as e:
+            raise RuntimeError(f"Error in langchain graph: {e}") from e
+
+    async def _astream(self, value: LanggraphWrapperInput) -> AsyncGenerator[LanggraphWrapperOutput, None]:
+        try:
+            async for output in self._graph.astream(value):
+                yield output
+        except Exception as e:
+            raise RuntimeError(f"Error in langchain graph: {e}") from e
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)